### DISCUSSION TITLE
predictions from evaluate.progressive_val_score

### DISCUSSION THREAD

**Discussion by 0r** [online-ml/river#578] [Q&A]
Is there a way to retrieve the y predictions generated with evaluate.progressive_val_score?

https://riverml.xyz/latest/api/evaluate/progressive-val-score/
> At each step the model is either asked to _predict an observation_, or is either updated

**Comment by MaxHalford** [comment]
Hello! I apologize for answering so late.

No currently that can't be done. Can I ask you would want to do this? A workaround could be to define your own `predict_one` method in your model and do whatever the hell you want with the prediction before returning it.
